src/main/java/mate/academy/controllers/OrderController.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.controllers;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.lib.Injector;
+import mate.academy.model.Product;
+import mate.academy.service.OrderService;
+import mate.academy.service.ShoppingCartService;
+import mate.academy.service.UserService;
+
+public class OrderController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+    private OrderService orderService =
+            (OrderService) INJECTOR.getInstance(OrderService.class);
+    private ShoppingCartService shoppingCartService =
+            (ShoppingCartService) INJECTOR.getInstance(ShoppingCartService.class);
+    private UserService userService =
+            (UserService) INJECTOR.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Product> products = shoppingCartService.getAll(shoppingCartService.get(USER_ID));
```

Comment:
```suggestion
        List<Product> products = shoppingCartService.getAll(shoppingCartService.getByUserId(USER_ID));
```
id юзера та корзини можуть не співпадати